### DISCUSSION TITLE
smartphone version

### DISCUSSION THREAD

**Discussion by ReySapoVII** [pear-devs/pear-desktop#2735] [Ideas]
is it posible to make a smartphone version for this?
i've been searching for days but i dont find any youtube music client that works properly, i had one but it stopped working, and in computer this is the only thing i found that works, so i wanted to know if there is any possibility to port this to phones.
I know there are other discussions about this but they are from 2021 and outdated

**Comment by spoonh** [comment]
https://apps.apple.com/us/app/youtube-music/id1017492454

**Reply by ReySapoVII** [reply]
no adblock, no background play, no extra features etc...

**Comment by ReySapoVII** [comment]
ok so i found in a discussion this app called ReVanced Manager, its on github too, you basically install youtube music normally and then with this app "patch" the youtube music app and it works for now

**Comment by apkgeek** [comment]
very nice tesla is releasing their [latest smart phone](https://teslapiphone.org) by tesla called as tesla pi phone
